scripts/build/windows.bat

```diff
@@ -226,6 +233,10 @@ signtool verify /pa /v %package_output%\Etcher.exe
 
 upx -9 %package_output%\*.dll
 
+cd %package_output%^
+ && 7z a -tzip ..\installers\%package_name%.zip *^
```

Comment:
I'm not sure how 'clean' those 3rd-party scripts are? Looks like some might work in some environments, and some in others.
Seems like just using 7zip might be more reliable?